### PR TITLE
DRAFT: Add cleanup filestore command

### PR DESCRIPTION
**Pull request by stefina**

### CODE REVIEW

File: ckanext/ogdchcommands/commands.py [hunk 1]
```diff
@@ -91,6 +97,11 @@ def __init__(self, name):
             default=30,
             help='Initial timeframe to keep harvested datasets, '
                  'jobs and objects.')
+        self.parser.add_option(
```
Comment (stefina) [inline ckanext/ogdchcommands/commands.py]: this should not be there

File: ckanext/ogdchcommands/commands.py [hunk 2]
```diff
@@ -39,9 +39,15 @@ class OgdchCommands(ckan.lib.cli.CkanCommand):
         paster ogdch cleanup_datastore
 
         # Cleanup resources
-        paster ogdch cleanup_resources
+        paster ogdch cleanup_resources [--dryrun]
```
Comment (stefina) [inline ckanext/ogdchcommands/commands.py]: dryrun does not work for now

File: ckanext/ogdchcommands/commands.py [hunk 3]
```diff
@@ -47,6 +61,11 @@ class OgdchCommands(ckan.lib.cli.CkanCommand):
         # - the command can be performed with a dryrun option where the
         #   database will remain unchanged
 
+        # Cleanup filestore
+        paster ogdch cleanup_filestore [--dryrun]
```
Comment (stefina) [inline ckanext/ogdchcommands/commands.py:65]: dryrun does not work yet

File: ckanext/ogdchcommands/commands.py [hunk 4]
```diff
@@ -266,6 +286,30 @@ def _get_datastore_table_page(self, context, offset=0):
...
+            context,
+            {
+                'dryrun': self.options.dryrun,
+            })
+        if self.options.dryrun:
+            print(msg_filestore_cleanup_dryrun
+                  .format(result.get('filepaths')))
+        else:
+            print(msg_filestore_cleanup
+                  .format(result.get('filepaths')))
```
Comment (bellisk) [inline ckanext/ogdchcommands/commands.py:310]: there should be another argument for `format()` here, probably `len(result.get.('filepaths))`

**Pull request closed without merging**